test/unit/run.test.js

```diff
@@ -282,25 +282,26 @@ describe('run module', function () {
             });
 
             it('should handle options.folder passed as string correctly', function (done) {
-                run({ collection: {}, folder: 'f1' }, function (options) {
-                    expect(options).to.have.deep.property('entrypoint', { execute: 'f1' });
+                run({ collection: {}, folder: 'f1', preserveOrder: false }, function (options) {
```

Comment:
Since preserveOrder is passed to the entrypoint. It would have to be changed in order to make the existing tests work.
This approach has been accepted by @codenirvana and merged in postman-runtime.